src/foam/u2/crunch/CapabilityFeatureView.js

```diff
@@ -119,7 +119,7 @@ foam.CLASS({
             return this.E()
               .start('', { tooltip: cjStatus.documentation })
                 .addClass(this.myClass('badge'))
-                .add(foam.u2.view.ReadOnlyEnumView.create({ data: cjStatus }))
+                .add(this.ReadOnlyEnumView.create({ data: cjStatus, showGlyph: true }))
```

Comment:
this was the issue where you had to use the require view in order to get the proper context within the nested view right?

Reply:
Yes